### DISCUSSION TITLE
Why do I lose EXIF ​​meta information after LZW compression of TIFF files

### DISCUSSION THREAD

**Discussion by Yuj1010** [ImageMagick/ImageMagick#1996] [General] [answered]
Use ImageMagick to compress TIFF files. 
command like this:
magic input.tif -compress LZW output.tif 
The exif information added in input.tif by exiftool is lost in output.tif, other information is still saved, such as xmp information, IPTC information
Why and how to solve it

**Comment by dlemstra** [comment, marked as answer]
The reason why this is happening can be found here: https://github.com/ImageMagick/ImageMagick/issues/1674
